software/contrib/preset_manager.py

```diff
@@ -0,0 +1,191 @@
+from time import sleep
+from europi import *
+from europi_script import EuroPiScript
+
+
+class PresetManager(EuroPiScript):
+    def __init__(self):
+        super().__init__()
+        state = self.load_state_json()
+
+        self.blank_preset = ["blank preset", [0,0,0,0,0,0]]
```

Comment:
I would pull this out into a function, `make_blank_preset(name)` that takes the name and returns the list of `[name, [0, 0, 0, 0, 0, 0]]` which you could then call wherever. Then you never have to worry about having typed in the right number of `0`s .

Reply:
This is a very good idea, I'll add it